Documentation/Signoffs/Motor System.md

```diff
@@ -0,0 +1,317 @@
+
+**The dividers on each platform:**
+* **Will also be made of acrylic**  
+
+* **Will be 1/16th inch thick (giving a density of 0.365 lbs/ sq. ft.)**  
+
+* **Will be large enough to encompass the whole board (8.5 x 8 inches, will assume 9 x 9 inches).**  
+^ There will be 20 dividers per platform, equating to 60 total platforms.  
+
+* **Given these values, each divider will weigh 0.27375 lbs (0.365 * 0.75). The total weight of the dividers will be 16.425 lbs.**  
+
+**The hollow cylinder supporting the platforms:**  
+* **Will be made of aluminum or pvc (assuming aluminum for worst-case)**  
+
+* **Will be 36- to 40-inches tall (assuming 40 inch for worst-case).**  
+
+* **Will be 1/16th or 1/8th thick (assuming 1/8th inch for worst-case).**  
+^ Given the density of aluminum (2.7 g/cm^3) and given the area equation for a hollow cylinder  
+
+$M = (π)(h)(mD)(r^2 - (r - t)^2)$  
+where:  
+  M = mass of the hollow cylinder
+  h = height  
+  r = outer radius  
+  t = thickness  
+  mD = Mean Density of cylinder material  
+The hollow aluminum cylinder will weigh **9.622 lbs**  
+
+**The Motor:**
+* **Given the calculations above, must rotate a total weight of 70.858 lbs. (We will assume roughly 20% tolerance for error, so for calculations we will use 85 lbs)**
+
+**Circuity:** 
+* **Will be protected at 125% the nameplate current rating (7.5A).** This comes from the following codes.
+
+The standard NEC 430.52(D) states "Torque motor branch circuits shall be protected at the motor nameplace current rating in accordance with 240.4(B)
+
+^ Source: https://up.codes/s/rating-or-setting-for-individual-motor-circuit
+
+NEC 240.4(B) is the standard for overcurrent devices rated 800 amperes or less, which allows the use of a higher rated overcurrent device as long as the conditions are met. All of these conditions have been cleared.  
+ * The conductors being protected are not part of a branch circuit supplying more than one receptacle for cord-and-plug-connected portable loads.  
+ * The ampacity of the conductors does not correspond with the standard ampere rating of a fuse or a circuit breaker without overload trip adjustments above its rating (but that shall be permitted to have other trip or rating adjustments).  
+ * The next higher standard rating selected does not exceed 800 amperes.  
+^ Source: https://up.codes/viewer/illinois/nfpa-70-2020/chapter/2/wiring-and-protection#240.4_(B)  
+
+The standard NEC 705.12(D)(2) Bus or Conductor Ampere Rating states "One hundred twenty-five percent [125%] of the inverter output circuit current shall be used in ampacity calculations for the following: Feeders, Taps, and Busbars."
+
+^ Source: https://up.codes/viewer/delaware/nfpa-70-2014/chapter/7/special-conditions#705.12_(D)_(2)
+
+## Schematic
+
+![Schematic](https://github.com/DillonSW/Capstone_Team_5/blob/Team5-signoff-Motor-System/images/DriverAndMotorRevisionTwo.jpg)
```

Comment:
Getting better! What is the purpose of the step down jump? It looks like the motor says it accepts 110VAC. Wall voltage is nominally considered to be 110VAC.

Reply:
The motor is controlled by the HSS86 driver, which the datasheet says can accept either 20-80VAC, or 30-110VDC. Because of this, we must step down the incoming VAC from the wall outlet.

Reply:
I see. That makes sense. I misread the 110 as VAC.